### DISCUSSION TITLE
No proper web report is generating

### DISCUSSION THREAD

**Discussion by forzaturian** [allure-framework/allure2#1431] [General Discussion]
Hi guys!
I'm running allure 2.13.8 with cypress and my last few reports are not generating properly.
Could you please advise on this issue?

Thank you!

![image](https://user-images.githubusercontent.com/47934778/134743909-27d61373-ee19-4c7e-b8b3-34708bc1ad50.png)

**Comment by Avaddon-tfa** [comment]
Hello there! At first sight, it seems that the directory for storing results broke. 
Please, make sure that, after the run, the allure-results folder gets created. Then generate a report only when it exists.

In case this issue is excluded, please, provide more details for investigation by our team.
Thank you!

**Comment by forzaturian** [comment]
Thanks for your response!

The thing is the alure-results folder has not been created.
here is the screenshot from my **package.json** :
![image](https://user-images.githubusercontent.com/47934778/135560103-8feb7116-7d8e-4cbe-8532-86e8c0982c0a.png)
I have specified the path to allure-results in **cypress.json**, but it didn't help
`{
    "viewportWidth": 1920,
    "viewportHeight": 1080,
    "video": false,
    "chromeWebSecurity": false,
    "defaultCommandTimeout": 10000,
    "allureResultsPath": "allure-results/results",
    "reporter": "mochawesome",
    "reporterOptions": {
        "html": false,
        "json": true,
        "reportDir": "cypress/reports",
        "overwrite": false
    }
}`

**Reply by Avaddon-tfa** [reply]
Thank you! We have a couple of guesses. Stay tuned, and we'll be back with a piece of advice!

**Reply by forzaturian** [reply]
> Thank you! We have a couple of guesses. Stay tuned, and we'll be back with a piece of advice!

any updates?

**Comment by forzaturian** [comment]
Hi @Avaddon-tfa !
Can you please help me with this issue?

**Comment by forzaturian** [comment]
![image](https://user-images.githubusercontent.com/47934778/163823735-e74d655e-2745-47a5-8d45-00ae26c76999.png)

**Comment by baev** [comment]
Closed as stale. For the future reference, please see https://allurereport.org/docs/how-it-works/
